ipn/ipnlocal/local.go

```diff
@@ -76,6 +76,7 @@ import (
 	"tailscale.com/net/packet"
 	"tailscale.com/net/tsaddr"
 	"tailscale.com/net/tsdial"
+	"tailscale.com/net/udprelay"
```

Comment:
Can you please structure this as its own package that can be conditionally registered (condregister) so it can be build-tagged away?

We've moving towards doing that to all existing features (http://go/min) 

See 1562a6f2f2b8017a65ae147e48f23e1ec113ac2f as a prior example.

/cc @danderson @creachadair

Reply:
closing in favor of https://github.com/tailscale/tailscale/pull/15699